tracer/test/Datadog.Trace.Tests/PublicApiTests.cs

```diff
@@ -26,4 +29,22 @@ public void ExposesTracingHttpModule()
         }
 #endif
     }
+
+    public class DatadogTraceAnnotationsTests : PublicApiTestsBase
+    {
+        public DatadogTraceAnnotationsTests()
+            : base(typeof(TraceAttribute).Assembly)
+        {
+        }
+    }
+
+#if NETFRAMEWORK
+    public class DatadogTraceAspNetTests : PublicApiTestsBase
+    {
+        public DatadogTraceAspNetTests()
+            : base(Assembly.Load("Datadog.Trace.AspNet"))
+        {
+        }
+    }
+#endif
```

Comment:
Is this one necessary? 🤔

Reply:
Removed the Datadog.Trace.AspNet api test in 9f1de3d71b9e8f0c5d669fc210ea1c24704d7e6e